api/nodes/serializers.py

```diff
@@ -1491,9 +1494,9 @@ class Meta:
         type_ = 'draft_registrations'
```

Comment:
Corbin's work will turn this to `draft-registrations` - will the new DraftSerializer need a new type??